ui/components/app/collectible-details/collectible-details.js

```diff
@@ -81,6 +81,7 @@ export default function CollectibleDetails({ collectible }) {
     imageOriginal ?? image,
     ipfsGateway,
   );
+  const collectibleImageAlt = description ?? `${name} Token ID: ${tokenId}`;
```

Comment:
To avoid using the same logic  in multiple files, we should create a helper file with a function called something like `getNftImageAlt` that creates this string.

Additionally, `Token ID` would nee to be localized.  Instead, maybe we do: `${name} #{tokenId}`